src/fsharp/SyntaxTree.fsi

```diff
@@ -0,0 +1,2110 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace rec FSharp.Compiler.Syntax
+
+open FSharp.Compiler.Syntax
+open FSharp.Compiler.Text
+
+/// Represents an identifier in F# code
```

Comment:
Could xml docs be removed from the fs file due to addition of the fsi one, so for each declaration there's a single place where the xml doc is?

Reply:
> Could xml docs be removed from the fs file due to addition of the fsi one, so for each declaration there's a single place where the xml doc is?

We have no hard and fast rule in the compiler - in some places we duplicate (e.g. NameResolution.fs) and in others we don't.

I think I'll leave it as is for now.

Reply:
I thought about it and agree with you - I removed all the comments from SyntaxTree.fs